package.json

```diff
@@ -26,6 +27,7 @@
     "@babel/plugin-transform-modules-commonjs": "^7.2.0",
     "@babel/preset-env": "^7.3.4",
     "@babel/preset-react": "^7.0.0",
+    "@types/react": "^16.8.5",
```

Comment:
Where does it depend on React types actually? I don't see any reference to it.

Reply:
It's necessary to use React hooks in typings test.
https://github.com/mpeyper/react-hooks-testing-library/pull/8/files#diff-e1558258b996f87650686aec9e520a38R1